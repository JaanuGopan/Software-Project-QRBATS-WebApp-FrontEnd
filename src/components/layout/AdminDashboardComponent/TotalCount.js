import React from "react";
import "../../../pages/AdminDashboard/AdminDashboard.css";

const TotalCount = ({ total, countIcon, countTitle }) => {
  return (
    <div className="inform">
      <div className="inform1">
        <p
          style={{
            color: "white",
            fontSize: "large",
            fontWeight: "bold",
            padding: "5%",
          }}
        >
          {total}
        </p>
        {countIcon}
      </div>
      <p
        className="informationText"
        style={{
          color: "white",
          fontWeight: "bold",
          padding: "5%",
        }}
      >
        {countTitle}
      </p>
    </div>
  );
};

export default TotalCount;
