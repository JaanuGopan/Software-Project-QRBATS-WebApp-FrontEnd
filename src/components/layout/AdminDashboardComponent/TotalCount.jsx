import React from 'react';
import 'src/pages/dashboard/AdminDashboard/AdminDashboard.jsx';

const TotalCount = ({ total, countIcon, countTitle }) => {
  return (
    <div className="inform">
      <div className="inform1">
        <p
          style={{
            color: 'white',
            fontSize: 'large',
            fontWeight: 'bold',
            padding: '5%',
          }}
        >
          {total}
        </p>
        {countIcon}
      </div>
      <p
        className="informationText"
        style={{
          color: 'white',
          fontWeight: 'bold',
          padding: '5%',
        }}
      >
        {countTitle}
      </p>
    </div>
  );
};

export default TotalCount;
