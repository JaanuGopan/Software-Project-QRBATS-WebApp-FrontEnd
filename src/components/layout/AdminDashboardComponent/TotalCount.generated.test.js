import renderer from 'react-test-renderer';
import React from "react";

import TotalCount from "./TotalCount";

jest.mock("../../../pages/AdminDashboard/AdminDashboard.css");

const renderTree = tree => renderer.create(tree);
describe('<TotalCount>', () => {
  it('should render component', () => {
    expect(renderTree(<TotalCount 
    />).toJSON()).toMatchSnapshot();
  });
  
});