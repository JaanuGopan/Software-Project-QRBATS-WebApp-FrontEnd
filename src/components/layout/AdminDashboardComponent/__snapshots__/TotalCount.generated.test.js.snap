// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TotalCount> should render component 1`] = `
<div
  className="inform"
>
  <div
    className="inform1"
  >
    <p
      style={
        Object {
          "color": "white",
          "fontSize": "large",
          "fontWeight": "bold",
          "padding": "5%",
        }
      }
    />
  </div>
  <p
    className="informationText"
    style={
      Object {
        "color": "white",
        "fontWeight": "bold",
        "padding": "5%",
      }
    }
  />
</div>
`;
